demo/main.css

```diff
@@ -1,3 +1,23 @@
 li.fs-api-directory-collapse > ul {
   display: none;
 }
+
+ul {
+  list-style-type: none;
+}
+
+.fs-api-directory-handler {
+  cursor: pointer;
+}
+.fs-api-directory > .fs-api-directory-handler {
+  display: inline-block;
+  transform: rotate(90deg);
+  -ms-transform: rotate(90deg);
```

Comment:
Remove `-ms-transform`. We can use it without the `-ms-` vendor prefix these days: https://caniuse.com/#feat=transforms2d.